db/file/file.go

```diff
@@ -54,6 +54,10 @@ func (db *DB) Connect(format, conn string, maxOpenConn int) error {
 	return nil
 }
 
+func (db *DB) Close() {
+	// nothing to do
```

Comment:
No, because there are no open file handles in file db.

For instance, have a look at any of mutating methods (like func (tx *Transaction) Update(resource *schema.Resource) error). First the file is read to memory (using db.load()), then the mutation takes place in-memory, and finally the file is written back (using db.write()).